code/modules/clothing/spacesuits/plasmamen.dm

```diff
@@ -84,12 +84,15 @@
 		set_light(0)
 
 	for(var/X in actions)
-		var/datum/action/A=X
+		var/datum/action/A = X
 		A.UpdateButtonIcon()
 
 /obj/item/clothing/head/helmet/space/plasmaman/extinguish_light()
+	var/mob/living/carbon/human/H = null
+	if(ishuman(loc))
+		H = loc
 	if(on)
-		toggle_light()
+		toggle_light(H)
```

Comment:
So you don't have to declare a human var and assign it, I recommend doing something like:
```dm
/obj/item/clothing/head/helmet/space/plasmaman/extinguish_light()
    if(!on)
        return
    if(ishuman(loc))
        toggle_light(loc)
    else
        toggle_light()
```
Or even something more compact like:
```dm
/obj/item/clothing/head/helmet/space/plasmaman/extinguish_light()
    if(!on)
        return
    toggle_light(ishuman(loc) ? loc : null)
```